src/main/java/org/codeforamerica/messaging/config/TrustedPortConfiguration.java

```diff
@@ -0,0 +1,27 @@
+package org.codeforamerica.messaging.config;
+
+import org.apache.catalina.connector.Connector;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.web.embedded.tomcat.TomcatServletWebServerFactory;
+import org.springframework.boot.web.server.WebServerFactoryCustomizer;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+
+@Configuration(proxyBeanMethods = false)
+public class TrustedPortConfiguration {
+    @Value("${server.trustedPort:null}")
+    private int trustedPort;
+
+
+    @Bean
+        public WebServerFactoryCustomizer<TomcatServletWebServerFactory> connectorCustomizer() {
+            return (tomcat) -> tomcat.addAdditionalTomcatConnectors(createConnector());
+        }
+
+        private Connector createConnector() {
+            Connector connector = new Connector("org.apache.coyote.http11.Http11NioProtocol");
```

Comment:
This connector class parameter is a little bit mysterious. Is it likely to ever change or be reconfigured?

Reply:
I agree. I doubt it will change. The only reference I have seen for it is https://docs.spring.io/spring-boot/docs/1.4.0.RELEASE/reference/html/howto-embedded-servlet-containers.html#howto-enable-multiple-connectors-in-tomcat.